apps/filecoin-site/src/app/_constants/siteMetadata.ts

```diff
@@ -3,6 +3,8 @@ import type { Metadata } from 'next'
 const BASE_DOMAIN = 'filecoin.io'
 const BASE_URL = `https://${BASE_DOMAIN}`
 const ORGANIZATION_NAME = 'Filecoin'
+const PARENT_ORGANIZATION_NAME = 'Filecoin Foundation'
+const PARENT_ORGANIZATION_URL = 'https://fil.org/'
```

Comment:
I'm not sure it's right or politically safe to call the foundation the parent organization..